packages/workspace/src/schematics/shared-new/shared-new.ts

```diff
@@ -125,13 +130,18 @@ export function sharedNew(cli: string, options: Schema): Rule {
 }
 
 function addDependencies(options: Schema) {
+  const nxCloudVersion = options.nxCloud ? { '@nrwl/nx-cloud': 'next' } : {};
+  const noPreset = options.nxCloud
```

Comment:
This could be a new rule altogether. It won't duplicate the install.

Reply:
Fixed it